help/destinations/catalog/personalization/pega-profile.md

```diff
@@ -0,0 +1,134 @@
+
+>[!IMPORTANT]
+>
+>This documentation page was created by Pegasystems. For any inquiries or update requests, please contact Pega directly [here](mailto:support@pega.com).
+
+## Use cases 
+
+To help you better understand how and when you should use the [!DNL  Pega Profile Connector] destination, here are sample use cases that Adobe Experience Platform customers can solve by using this destination.
+
+### Use case 1
+
+A marketer wants to initially setup Pega Customer Decision Hub with profile data loaded from Adobe Experience Platform. This is an initial full load followed by delta loads on a scheduled basis.
+
+### Use case 2
+
+A marketer wants up-to-date profile data from Adobe Experience Platform available in Pega Customer Decision Hub that enhances the Pega insights around customer profiles on an ongoing basis.
+
+## Prerequisites {#prerequisites}
+
+Before you can use this destination to export data out of Adobe Experience Platform and import profiles into Pega [!DNL Pega Customer Decision Hub], make sure you complete the following prerequisites:
+
+* Configure [!DNL Amazon S3] bucket and the folder path to be used for export and import of data fles.
+* Configure the [!DNL Amazon S3] access key and [!DNL Amazon S3] secret key: In [!DNL Amazon S3], generate an `access key - secret access key` pair to grant Platform access to your [!DNL Amazon S3] account.
+* To successfully connect and export data to your [!DNL Amazon S3] storage location, create an Identity and Access Management (IAM) user for [!DNL Platform] in [!DNL Amazon S3] and assign permissions such as `s3:DeleteObject`, `s3:GetBucketLocation`, `s3:GetObject`, `s3:ListBucket`, `s3:PutObject`, `s3:ListMultipartUploadParts`
+* Make sure your [!DNL Pega Customer Decision Hub] instance is upgraded to 8.8 version or higher. 
+
+## Supported identities {#supported-identities}
+
+[!DNL Pega Customer Decision Hub] supports the activation of custom user IDs described in the table below. For more details, see [identities](/help/identity-service/namespaces.md).
+
+|Target Identity|Description|
+|---|---|
+|*CustomerID*|Common User Identifier that uniquely identifies a profile in [!DNL Pega Customer Decision Hub] and Adobe Experience Platform|
+
+{style="table-layout:auto"}
+
+## Export type and frequency {#export-type-frequency}
+
+Refer to the table below for information about the destination export type and frequency.
+
+| Item | Type | Notes |
+---------|----------|---------|
```

Comment:
missing pipe character at beginning of line. this will lead to a broken table.

Reply:
Added